server/eclipse-project/pom.xml

```diff
@@ -233,5 +233,10 @@
 			<artifactId>commons-lang3</artifactId>
 			<version>3.6</version>
 		</dependency>
+    <dependency>
+      <groupId>javax.xml.bind</groupId>
+      <artifactId>jaxb-api</artifactId>
+      <version>2.3.0</version>
+    </dependency>
```

Comment:
Does this get used in this PR?

Reply:
That was a weird quirk, idk, but it fixes an error I was seeing "javax.xml.bind not on classpath"